### PR TITLE
docs(play): mention limitation for execution listeners

### PR DESCRIPTION
**Pull request by huygur**
## Description

Execution listeners are currently not supported in Play. Until we fix this, we will add a note mentioning this limitation.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 1]
```diff
@@ -153,6 +153,8 @@ For more information about terms, refer to our [licensing and terms page](https:
 
 Although Play is compatible with cluster versions 8.5.1 and above, we fully support and recommend using versions 8.6.0 or higher.
 
+Play currently doesn't support execution listeners. As a workaround, you can skip the element using [modifications](#modify-a-process-instance).
```
Comment (mesellings) [inline docs/components/modeler/web-modeler/play-your-process.md:156]:
```suggestion
**Execution listeners:** Play does not currently support [execution listeners](/components/concepts/execution-listeners.md). As a workaround, you can skip the element using [modifications](#modify-a-process-instance).
```
Suggestion to make it more scannable, and link to execution listeners.

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 2]
```diff
@@ -153,6 +153,8 @@ For more information about terms, refer to our [licensing and terms page](https:
 
 Although Play is compatible with cluster versions 8.5.1 and above, we fully support and recommend using versions 8.6.0 or higher.
```
Comment (mesellings) [inline docs/components/modeler/web-modeler/play-your-process.md:154]:
```suggestion
**Version compatibility:** Although Play is compatible with cluster versions 8.5.1 and above, Camunda fully supports and recommends using versions 8.6.0 or higher.
```
Suggestion to align with change below for better scannability on the page.

File: versioned_docs/version-8.6/components/modeler/web-modeler/play-your-process.md [hunk 3]
```diff
@@ -153,6 +153,8 @@ For more information about terms, refer to our [licensing and terms page](https:
 
 Although Play is compatible with cluster versions 8.5.1 and above, we fully support and recommend using versions 8.6.0 or higher.
 
+Play currently doesn't support execution listeners. As a workaround, you can skip the element using [modifications](#modify-a-process-instance).
```
Comment (mesellings) [inline versioned_docs/version-8.6/components/modeler/web-modeler/play-your-process.md:156]:
```suggestion
**Execution listeners:** Play does not currently support [execution listeners](/components/concepts/execution-listeners.md). As a workaround, you can skip the element using [modifications](#modify-a-process-instance).
```
Backport the change suggested above

File: versioned_docs/version-8.6/components/modeler/web-modeler/play-your-process.md [hunk 4]
```diff
@@ -153,6 +153,8 @@ For more information about terms, refer to our [licensing and terms page](https:
 
 Although Play is compatible with cluster versions 8.5.1 and above, we fully support and recommend using versions 8.6.0 or higher.
```
Comment (mesellings) [inline versioned_docs/version-8.6/components/modeler/web-modeler/play-your-process.md:154]:
```suggestion
**Version compatibility:** Although Play is compatible with cluster versions 8.5.1 and above, Camunda fully supports and recommends using versions 8.6.0 or higher.
```
Backport the change suggested above

**Review by mesellings — DISMISSED** [review]
Lgtm 👍 - just some suggestions to make it easier to scan the page and see what this limitation concerns. Also, should we also add this limitation to the execution listeners page as well, as that may be the entry point for readers, who might not come into this via Play? Just a suggestion 🤷

**Comment by huygur** [comment]
@mesellings thanks for the review! I applied the suggestions.

Play is typically not cross-referenced on other pages with phrases like, 'you can try out process modifications in Play.' Therefore, I don’t think it’s necessary to mention the limitations on those pages as well.

**Review by mesellings — APPROVED** [review]
Lgtm 👍

**Pull request merged**